pkg/relayer/cmd/flags.go

```diff
@@ -0,0 +1,9 @@
+package cmd
+
+const (
+	FlagRelayMinerConfigPath = "config"
```

Comment:
```suggestion
	FlagRelayMinerConfigPath = "relay-miner-config"
```

Since we'll have an `app-gate-config` as well

Reply:
I think the flag should stay as `--config`, the corresponding key will be unique and unambiguous.